opyn/opyn/config.py

```diff
@@ -76,14 +77,17 @@ def validate_bid(
         sell_amount: int,
         buy_amount: int,
         referrer: str,
-        v: int,
-        r: str,
-        s: str,
+        signature: str,
         **kwargs,
     ) -> str:
         """Validate the signing bid"""
+        r, s, v = break_evm_signature_into_components(signature)
 
-        config = ContractConfig(address=contract_address, chain_id=chain_id, rpc_uri=rpc_uri)
+        config = ContractConfig(
+            address=contract_address,
+            chain_id=Chains(chain_id),
```

Comment:
Thank you for this! The same fix is also included in https://github.com/tradeparadigm/sdks/pull/40, probably we want to complete the mypy integration asap, a lot of fixes are waiting there